src/test/java/de/bonndan/nivio/ArchTest.java

```diff
@@ -0,0 +1,22 @@
+package de.bonndan.nivio;
+
+import com.tngtech.archunit.core.domain.JavaClasses;
+import com.tngtech.archunit.core.importer.ClassFileImporter;
+import com.tngtech.archunit.core.importer.ImportOption;
+import org.junit.jupiter.api.Test;
+
+import static com.tngtech.archunit.library.GeneralCodingRules.NO_CLASSES_SHOULD_ACCESS_STANDARD_STREAMS;
+
+class ArchTest {
+    @Test
+    void do_not_access_STD_streams() {
+        JavaClasses classes = new ClassFileImporter()
+                .withImportOption(ImportOption.Predefined.DO_NOT_INCLUDE_JARS)
+                .withImportOption(ImportOption.Predefined.DO_NOT_INCLUDE_TESTS)
+                .importClasspath();
```

Comment:
You could move the classes into a test attribute, even a static one.

Usually, the classes are used by many (if not all) ArchUnit tests and collecting the data from byte code is expensive.

Reply:
I took over to do this to merge this PR. I need ArchUnit for Issue 317!